inspire/modules/authors/recordext/fields/author.cfg

```diff
@@ -0,0 +1,363 @@
+    +---------+---------------------+
+    | 100__b  | Jr.                 |
+    +---------+---------------------+
+    | 100__c  | Sir                 |
+    +---------+---------------------+
+    | 100__d  | 1943-2001           |
+    +---------+---------------------+
+    | 100__g  | active              |
+    +---------+---------------------+
+    | 100__q  | Johnny Smith        |
+    +---------+---------------------+
+    """
+
+    schema:
+        {'name': {'type': 'dict', 'required': True,
+         'schema': {'last': {'type': 'string', 'empty': False},
+                    'first': {'type': 'string', 'empty': False},
+                    'numeration': {'type': 'string', 'nullable': True,
+                                   'regex': __import__('flask').current_app.config['INSPIRE_NAME_NUMERATIONS']},
+                    'title': {'type': 'string', 'nullable': True,
+                              'allowed': __import__('flask').current_app.config['INSPIRE_NAME_TITLES']},
+                    'birth_year': {'type': 'integer', 'nullable': True,
+                                   'min': __import__('flask').current_app.config['INSPIRE_YEAR_MIN'],
+                                   'max': __import__('flask').current_app.config['INSPIRE_YEAR_MAX']},
+                    'death_year': {'type': 'integer', 'nullable': True,
+                                   'min': __import__('flask').current_app.config['INSPIRE_YEAR_MIN'],
+                                   'max': __import__('flask').current_app.config['INSPIRE_YEAR_MAX']},
+                    'previous_name': {'type': 'string', 'nullable': True,
+                                      'empty': False},
+                    'status': {'type': 'string', 'nullable': True,
+                               'allowed': __import__('flask').current_app.config['INSPIRE_NAME_STATUSES']},
+                    'preferred_name': {'type': 'string', 'nullable': True,
+                                       'empty': False}}}}
+    creator:
+        marc, "100__", {'last': util_split(value['a'], ',', 0),
+                        'first': util_split(value['a'], ',', 1),
+                        'numeration': value['b'],
+                        'title': value['c'],
+                        'birth_year': int_util_split(value['d'], '-', 0),
+                        'death_year': int_util_split(value['d'], '-', 1),
+                        'previous_name': value['i'],
+                        'status': value['g'], 'preferred_name': value['q']}
+    producer:
+        json_for_marc(), {"100__a": "last + ', ' + first",
+                          "100__b": "numeration",
+                          "100__c": "title",
+                          "100__d": "birth_year + '-' + death_year",
+                          "100__g": "status", "100_q": "preferred_name"}
+
+emails:
+    """Contains email address.
+
+    Must be in form of `<name>@<domain>.<ending>`.
+
+    TO DO: define a type and regex for elements of the list.
```

Comment:
I think this should be available in general as a central validator. We always have the need for valid emails...